tensorflow/python/eager/benchmarks_test.py

```diff
@@ -1620,5 +1627,42 @@ def benchmark_tf_range_return_int32_GPU(self):
   def benchmark_tf_range_return_int64_GPU(self):
     self._benchmark_tf_range_return(dtype=dtypes.int64, device=GPU)
 
+  def _benchmark_embedding_lookup_sparse_sparse_input(self, batch_size=32000, device=GPU):
```

Comment:
Nit: what about changing the name to `_benchmark_embedding_lookup_sparse_with_sparse_input`?